gst/tensor_filter/tensor_filter.c

```diff
@@ -350,17 +354,17 @@ gst_tensor_filter_fix_caps (GstTensor_Filter * filter, gboolean isInput,
   staticcap = gst_static_caps_get (&rawcap);
 
   if (isInput == TRUE) {
-    type = prop->inputType;
+    type = prop->inputMeta.types;
     int i;
     for (i = 0; i < NNS_TENSOR_SIZE_LIMIT; i++) {
-      dimension[i] = prop->inputDimension[i];
+      dimension[i] = prop->inputMeta.dims[i];
```

Comment:
@myungjoo 
I agree with memcpy.
Anyway, the value of `prop` should be changed if `dimension` is modified?
Since I didn't care about the modification case, if it is, I need to implement that logic also.

Reply:
If it's read-only. We may keep swallow-copy. (To ensure readonlyness, I'd recommend to put "const" to the declaration though)

Reply:
Since there is no case to change `dimension` so far, read-only is not weird. However, it could be possible whenever we add dimension adjustment.